### DISCUSSION TITLE
Can the JDT URI be excluded from Javadoc?

### DISCUSSION THREAD

**Discussion by CarlosLlaque** [mfussenegger/nvim-jdtls#649] [Q&A]
Is it possible to configure jdtls so that it doesn't display JDT URIs in Javadoc?
![image](https://github.com/mfussenegger/nvim-jdtls/assets/64855159/8d8455a5-3c65-41b8-aaa4-47934c3dc649)

**Comment by mfussenegger** [comment]
In neovim nightly/0.10 they should already be concealed by default. For neovim 0.9 or earlier you can customize the syntax, see https://github.com/mfussenegger/nvim-jdtls/discussions/124#discussioncomment-1968776 

Given that this would also apply to other language servers, nvim-jdtls doesn't do anything like that by default
